src/cmd/release.go

```diff
@@ -45,6 +46,28 @@ var gitlabCmd = &cobra.Command{
 	},
 }
 
+// githubCmd represents the github command
+var githubCmd = &cobra.Command{
+	Use:   "github flavor",
+	Short: "Create a tag and release on GitHub based on flavor",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		releaserConfig, err := utils.LoadReleaserConfig(releaserDir)
+		if err != nil {
+			return err
+		}
+
+		currentFlavor, err := utils.GetFlavorConfig(args[0], releaserConfig)
+		if err != nil {
+			return err
+		}
+
+		rootCmd.SilenceUsage = true
+
+		return github.TagAndRelease(currentFlavor, tokenVarName)
+	},
+}
+
```

Comment:
This could be simplified if there was an interface that had the `TagAndRelease` function in it - https://golangdocs.com/interfaces-in-golang - this would also allow for easier future expansion and less copied code